### PR TITLE
[JitBackend] Implement simple binary ops

### PR DESCRIPTION
**Pull request by StrongerXi**
### Summary

"Parse" simple binary ops (add, sub, mul, div) in JIT.

### Test Plan (required)
Unit tests are added.

```
make JitTensorTest
./flashlight/fl/test/JitTensorTest
```

### CODE REVIEW

**Comment by StrongerXi** [comment]
Rebase.

**Comment by StrongerXi** [comment]
Rebase.

File: flashlight/fl/test/tensor/jit/JitTensorTest.cpp [hunk 1]
```diff
@@ -5,24 +5,63 @@
...
+class JitTensorTest : public ::testing::Test {
+ protected:
+  void SetUp() override {
+    fl::setDefaultTensorType<JitTensor<ArrayFireTensor>>();
+  }
+};
+
+template <typename Op>
+void testBinaryOp(Op func, BinaryOp op) {
   fl::setDefaultTensorType<JitTensor<ArrayFireTensor>>();
```
Comment (jacobkahn) [inline flashlight/fl/test/tensor/jit/JitTensorTest.cpp:38]: can be removed with fixture?

**Review by jacobkahn — APPROVED** [review]
Nice -- clean!

**Comment by StrongerXi** [comment]
Remove redundant code.

**Comment by StrongerXi** [comment]
Rebase.

**Pull request closed without merging**